addons/resource/models/resource.py

```diff
@@ -87,6 +93,15 @@ def _get_default_attendance_ids(self):
         'resource.calendar.leaves', 'calendar_id', 'Global Leaves',
         domain=[('resource_id', '=', False)]
         )
+    hours_per_day = fields.Float("Average hour per day", default=HOURS_PER_DAY, help="Average hours per day a resource is supposed to work with this calendar.")
+
+    @api.onchange('attendance_ids') 
+    def _onchange_hours_per_day(self):
+        attendances = self.attendance_ids.filtered(lambda attendance: not attendance.date_from and not attendance.date_to)
+        hour_count = 0.0
+        for attendance in attendances:
+            hour_count += attendance.hour_to - attendance.hour_from
+        self.hours_per_day = hour_count / float(len(set(attendances.mapped('dayofweek'))))
```

Comment:
Won't we have 7.3455987987 hour per day depending on attendances ? aka, rounding ?